packages/server-web/src/client/component/TodoText.vue

```diff
@@ -23,8 +23,19 @@ export default {
     }
   },
   methods: {
+    cancel() {
+      if (this.$props.visible) {
+        this.$data.todoTitle = this.$props.initialTodoTitle;
+        this.$emit('cancel');
+      }
+    },
+    escapeKeyListener(event) {
+      if (event.keyCode === 27) {
+        this.cancel();
+      }
+    },
     change(event) {
-      if(event.target.value !== "") {
+      if(event.target.value !== "" && event.type === "keyup") {
```

Comment:
Looks like formatter did not run on this file? `if (event.target.value ...`

When a blur event is called, this would not run. Do you want to have blur cancel or save the the todo?

Reply:
Looking at the "Changes", you likely want to save:
> - Adds the option to click somewhere else while inputing text. Changes will be saved

Because of `event.type === "keyup"`, this would not save, I suppose?

Reply:
yes, you're right